common/src/main/java/com/revenuecat/purchases/common/responses/CustomerInfoResponseJsonKeys.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.common.responses
+
+object CustomerInfoResponseJsonKeys {
+    const val REQUEST_DATE = "request_date"
+    const val REQUEST_DATE_MS = "request_date_ms"
+    const val SUBSCRIBER = "subscriber"
+    const val ORIGINAL_APP_USER_ID = "original_app_user_id"
+    const val ORIGINAL_APPLICATION_VERSION = "original_application_version"
+    const val ENTITLEMENTS = "entitlements"
+    const val FIRST_SEEN = "first_seen"
+    const val ORIGINAL_PURCHASE_DATE = "original_purchase_date"
+    const val NON_SUBSCRIPTIONS = "non_subscriptions"
+    const val SUBSCRIPTIONS = "subscriptions"
+    const val MANAGEMENT_URL = "management_url"
+    const val BILLING_ISSUES_DETECTED_AT = "billing_issues_detected_at"
```

Comment:
Looks like this was only replaced in the `OfflineCustomerInfoCalculator` but it's also used in `EntitlementInfoFactories`, we should also replace it there?